.github/workflows/cd.yml

```diff
@@ -69,6 +88,7 @@ jobs:
 
   release:
     name: Release
+    if: ${{ steps.calculate-release-flags.outputs.release-detected == 'true' }}
     needs: binaries
```

Comment:
```suggestion
    needs:
      - binaries
      - needs: calculate-release-flags
```

Reply:
I wrote that way because the `calculate-release-flags` is an indirect dependency via the `binaries`. But it won't work with an if expression above: `if: ${{ needs.calculate-release-flags.outputs.release-detected == 'true' }}`, so fixed as well!